database/sql/init.sql

```diff
@@ -1,7 +1,7 @@
 # Create the unit tests DB.
 CREATE DATABASE IF NOT EXISTS `wordpress_unit_tests`;
 GRANT ALL PRIVILEGES ON `wordpress_unit_tests`.* TO 'wp'@'localhost' IDENTIFIED BY 'wp';
-GRANT ALL PRIVILEGES ON `wordpress_unit_tests`.* TO 'wp'@'192.168.50.1' IDENTIFIED BY 'wp';
+GRANT ALL PRIVILEGES ON `wordpress_unit_tests`.* TO 'wp'@'192.168.56.1' IDENTIFIED BY 'wp';
```

Comment:
could these lines co-exist rather than requiring an IP change? Is there a specific reason the IP is needed? E.g. for external use?

Reply:
to be honest i dont know what should be the correct one here, i think the only reason i changed it is we changed the default private ip network here: https://github.com/Varying-Vagrant-Vagrants/VVV/commit/39ee551a9d6a6cb15699ca989cf8d706fbb6f52c

adding both / co-exists also fine
```
GRANT ALL PRIVILEGES ON `wordpress_unit_tests`.* TO 'wp'@'192.168.50.1' IDENTIFIED BY 'wp';
GRANT ALL PRIVILEGES ON `wordpress_unit_tests`.* TO 'wp'@'192.168.56.1' IDENTIFIED BY 'wp';
```
not sure if thats the _correct_ thing though.

external use, no. and its not required by / related with docker

Reply:
Ah in that case lets kill the `50.1` line, but I wouldn't expect local loopback to go over that network inside the container